demos/text-field.html

```diff
@@ -275,60 +271,71 @@ <h2>Text Field box</h2>
 
       <section class="example">
         <h2>Text Field - Leading/Trailing icons</h2>
-        <div class="demo-note">
-          <em>
-            Note: Some implementations utilizing leading and trailing icons may wish to set a width on the
-            <code>input</code> element to achieve uniform widths on text fields.
-            This demo achieves that by adding a custom class name: <code>custom-text-field-input</code> and setting
-            some style:
-          </em>
-          <br />
-          <!--
-            The following text formatting achieves the visual output we are aiming for
-            since the <pre> tag honors whitespace and line breaks.
-          -->
-          <pre>
-<code>.custom-text-field-input {
-  width: 300px;
-}</code></pre>
-        </div>
         <div id="demo-tf-box-leading-wrapper">
           <div id="tf-box-leading-example"
             class="mdc-text-field mdc-text-field--box mdc-text-field--with-leading-icon" data-demo-no-auto-js>
             <i class="material-icons mdc-text-field__icon" tabindex="0">event</i>
-            <input type="text" id="tf-box-leading" class="mdc-text-field__input custom-text-field-input">
+            <input type="text" id="tf-box-leading" class="mdc-text-field__input">
             <label for="tf-box-leading" class="mdc-text-field__label">Your name</label>
             <div class="mdc-text-field__bottom-line"></div>
           </div>
         </div>
         <div id="demo-tf-box-trailing-wrapper">
           <div id="tf-box-trailing-example"
             class="mdc-text-field mdc-text-field--box mdc-text-field--with-trailing-icon" data-demo-no-auto-js>
-            <input type="text" id="tf-box-trailing" class="mdc-text-field__input custom-text-field-input">
+            <input type="text" id="tf-box-trailing" class="mdc-text-field__input">
             <label for="tf-box-trailing" class="mdc-text-field__label">Your other name</label>
             <i class="material-icons mdc-text-field__icon" tabindex="0">delete</i>
             <div class="mdc-text-field__bottom-line"></div>
           </div>
         </div>
+        <div id="demo-tf-outlined-leading-wrapper">
+          <div id="tf-outlined-leading-example"
+            class="mdc-text-field mdc-text-field--outlined mdc-text-field--with-leading-icon" data-demo-no-auto-js>
+            <input type="text" id="tf-outlined-leading" class="mdc-text-field__input">
+            <label for="tf-outlined-leading" class="mdc-text-field__label">Your other name</label>
+            <i class="material-icons mdc-text-field__icon" tabindex="0">event</i>
```

Comment:
This element should be before the input and label in order for tab order to make sense (focus should move to icon and then text field for leading icon).